.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,63 @@
+name: Continuous Integration
+on: [push, pull_request]
+jobs:
+  build:
+    name: HHVM ${{matrix.hhvm}} - ${{matrix.os}}
+    strategy:
+      # Run tests on all OS's and HHVM versions, even if one fails
+      fail-fast: false
+      matrix:
+        os: [ ubuntu , macos ]
+        hhvm:
+          - 4.62
+          - latest
+          - nightly
+    runs-on: ${{matrix.os}}-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Composer
+        run: .github/workflows/install-composer.sh --install-dir=${{runner.temp}}
+      - name: Install HHVM (apt)
+        if: matrix.os == 'ubuntu'
+        run: |
+          set -ex
+          export DEBIAN_FRONTEND=noninteractive
+          sudo apt-get update
+          sudo apt-get install -y software-properties-common apt-transport-https
+          sudo apt-key add .github/workflows/hhvm.gpg.key
+          if [ "${{matrix.hhvm}}" = "nightly" ]; then
+            sudo add-apt-repository https://dl.hhvm.com/ubuntu
+            sudo apt-get install -y hhvm-nightly
+          elif [ "${{matrix.hhvm}}" = "latest" ]; then
+            sudo add-apt-repository https://dl.hhvm.com/ubuntu
+            sudo apt-get install -y hhvm
+          else
+            DISTRO=$(lsb_release --codename --short)
+            sudo add-apt-repository \
+              "deb https://dl.hhvm.com/ubuntu ${DISTRO}-${{matrix.hhvm}} main"
+            sudo add-apt-repository https://dl.hhvm.com/ubuntu
```

Comment:
Would it work if you switched the order of these `add-apt-repository` calls?

Then you could pull the "https://dl.hhvm.com/ubuntu" one out of the if statement completely

Reply:
This is a bug; only the first one can be here.

This is currently invisible as this projects' minimum currently equals latest

Reply:
when 4.63 is released next week:
- the first line here would expose 4.62
- the second line would expose 4.63
- `apt-get install hhvm` would install 4.63